lib/reevoo_mark_utils.php

```diff
@@ -30,7 +30,10 @@ function getTrkref($options = array()) {
   }
 
   function getPaginationParams($options = array()) {
-    $page = self::presenceKey($_GET, 'reevoo_page', 1);
+    $page = isset($options['page'])
```

Comment:
doen't look like good idea to me ... site can use `page` param for it's own context ... at least `reevoo_page` should have a higher priority than `page`